docs/DeveloperGuide.md

```diff
@@ -1476,6 +1476,62 @@ testers are expected to do more *exploratory* testing.
 
    1. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the list size)<br>
       Expected: Similar to previous.
+ 
+### Updating a Product in a Supplier/Warehouse
+
+1. Update command format: `update ct/TYPE n/NAME pd/PRODUCT_NAME [q/QUANTITY] [t/TAG…​]`
```

Comment:
Oops prefix should be index instead of name here!

Reply:
Thanks! will fix